SpywareFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||webapi.tresorit.com/*/weblogformdata/*
```

Comment:
There is unnecessary `*` at the end,
Btw, I'd also add `||tresorit.com/page-init-tracking.*.js`.